js/index.js

```diff
@@ -0,0 +1,99 @@
+// Data:
+
+const answers = {
+  // TODO(Baryska): Consider adding more response options e.g. uncapitalized versions of responses or responses with common typos.
+  1: ["2021"],
+  2: ["канатоходец", "канатоходцы"],
+  3: ["1Е2Д3Г4В5Б6А", "1Е 2Д 3Г 4В 5Б 6А"],
+  4: ["Лев", "лев"],
+  5: ["Эжен", "Делакруа", "Эжен Делакруа", "Делакруа Эжен"],
+  6: ["17"],
+  7: ["Арколе", "Арколь"],
```

Comment:
Here you could add more options on what are valid responses for different questions.